docs(review): docs review MTA-2766 (#1285)
scaleway/docs-content: compute/instances/troubleshooting/cant-connect-ssh.mdx

In this case, the Instance rejected your attempt to connect. There is most probably a problem with the SSH key pair - either the public key on the Instance or the private key on your local machine, or both.

<Message type="tip">
  Keep in mind that you [can not connect](/faq/kubernetes/#why-cannot-i-connect-to-my-nodes-using-ssh) via SSH to Instances that are managed by Kubernetes Kapsule. You can manage these Instances using Kubernetes tools directly.
</Message>

### Check that the Instance has the correct public key

> When you create your Instance, be sure to check at step 6 of the creation wizard that you are uploading a public key to which you have the corresponding private key on your local machine:

Reviewer: ```suggestion
When you create your Instance, check step 6 of the creation wizard and make sure you are uploading a public key to which you have the corresponding private key on your local machine:
``

> When you create your Instance, be sure to check at step 6 of the creation wizard that you are uploading a public key to which you have the corresponding private key on your local machine:

Reviewer: ```suggestion
When you create your Instance, check step 6 of the creation wizard and make sure you are uploading a public key to which you have the corresponding private key on your local machine:
```
Maybe it's clearer that way syntax-wise?
